charts/finrem-cos/values.yaml

```diff
@@ -13,8 +13,4 @@ java:
   keyVaults:
     finrem:
       secrets:
-        - finrem-doc-s2s-auth-secret
-        - idam-secret
-        - username-solicitor
```

Comment:
@johnoneill46 from quick look, functional tests won't pass without these:
```
@Value("${idam.username}")
    private String idamUserName;
```

Reply:
Functional tests don't need these to be defined in application. They are defined in Jenkinsfile_CNP. see https://github.com/hmcts/cnp-jenkins-library#secrets-for-functional--smoke-testing